code/modules/mob/camera/imaginary_friend.dm

```diff
@@ -137,6 +137,8 @@
 	med_squad_mobhud ? H.add_hud_to(src) : H.remove_hud_from(src)
 	H = GLOB.huds[DATA_HUD_SQUAD_REBEL]
 	med_squad_mobhud ? H.add_hud_to(src) : H.remove_hud_from(src)
+	H = GLOB.huds[DATA_HUD_SQUAD_SOM] //what is this admemery?
+	med_squad_mobhud ? H.add_hud_to(src) : H.remove_hud_from(src)
```

Comment:
?

Reply:
I don't really know what this does because it's some admin thing, so I just copy pasted it for the SOM hud in case it's actually used for something.